CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Changelog
+
+Nock's changelog is included directly in [the GitHub release messages](https://github.com/nock/nock/releases). These are automatically generated by commit messages which follow [semantic-release](https://github.com/semantic-release/semantic-release), and released by [@nockbot](https://github.com/nockbot). For notes on specific versions, please go there.
```

Comment:
I’d maybe rephrase it a little bit

> Nock’s changelog can be found directly in the [GitHub release notes](https://github.com/nock/nock/releases). These are automatically created by [semantic-release](https://github.com/semantic-release/semantic-release) based on three [commit message conventions](https://semantic-release.gitbook.io/semantic-release#commit-message-format)

I don’t think we need to mention @nockbot here. I feel that it’s an internal detail that is not to interest to people looking for the changelog